WordPress/src/main/java/org/wordpress/android/util/image/getters/GlideImageGetter.kt

```diff
@@ -0,0 +1,74 @@
+package org.wordpress.android.util.image.getters
+
+import android.content.Context
+import android.graphics.drawable.Drawable
+import android.text.Html
+import android.widget.TextView
+import com.bumptech.glide.Glide
+import org.wordpress.android.WordPress
+import org.wordpress.android.util.PhotonUtils
+import org.wordpress.android.util.R
+import org.wordpress.android.util.image.ImageManager
+import org.wordpress.android.util.image.ImageType
+import java.lang.ref.WeakReference
+import java.util.HashSet
+import javax.inject.Inject
+
+class GlideImageGetter(
```

Comment:
Can we please add some comments about what this class is for? We don't have anything about it in the PR description either, so I don't know how to review this code. It'd be great if you can add why it's necessary and what it does, thanks!

Reply:
Also, I am curious if we can find a better name for the class. I don't know what it does yet, but since it starts with `Glide` I just assumed it was part of the 3rd party library when I saw its usage.